src/libstd/io/stdio.rs

```diff
@@ -872,6 +872,84 @@ impl fmt::Debug for StderrLock<'_> {
     }
 }
 
+/// Locks the current handle and reads a line of input, returning a `String` containing
+/// the input.
+///
+/// If you need more explicit control over
+/// locking, see the [`Stdin::lock`] and [`Stdout::lock`] methods.
+///
+/// [`Stdin::lock`]: struct.Stdin.html#method.lock
+/// [`Stdout::lock`]: struct.Stdout.html#method.lock
+///
+/// ### Note: Safety and Usage
+/// This function is not the recommended solution to get user input, it should
+/// only be used for simple actions. If you require more explicit control over
+/// getting user input, see the [`Stdin::read_line`] method.
+/// 
+/// [`Stdin::read_line`]: struct.Stdin.html#method.read_line
+///
+/// # Examples
+///
+/// ```rust
+/// use std::io::input_prompt;
+///
+/// fn main() {
+///     let user_input = input_prompt("Please enter some text: ");
+///         
+///     println!("You typed: {}", user_input);
+///         
+/// }
+/// ```
+#[stable(feature = "input_prompt", since = "1.44.1")]
```

Comment:
These functions should start out as unstable, *not* stable.

Reply:
Made both functions unstable